File: python_code/temperature.py
#Author : Daniel Kareti
#Modified on 3/18/25

#reading input from user in python

temp = int(input("Enter a temperature in degrees Fahrenheit: "))

temp_celsius = (int(temp) - 32) * float(5/9)

temp_kelvin = temp_celsius + 273.15

#round values to two decimal places
temp_celsius = round(temp_celsius, 2)
temp_kelvin = round(temp_kelvin, 2)

print("Your given temperature is " + temp + ". This converted into Celsius and Kelvin is the following:\n")
print("     Celsius: " + str(temp_celsius))
print("     Kelvin: " + str(temp_kelvin)) 

File: c_code/temperature.c
//User input and output in C
//Daniel Kareti
//Created on: 08/12/22


#include <stdio.h>
int main(){
    int temp;
    /* The following two lines are how to get user input in C */
    printf("Enter a temperature in degrees Fahrenheit: " );
    scanf("%d", &temp);
    
    double celsius_temp = ((temp - 32) * (5.0/9.0));
    
    double kelvin_temp = (celsius_temp + 273.15);

    printf( "The given temperature was %d. This converted into Celsius and Kelvin is:\n", temp);
    printf("    Celsius: %.2lf\n    Kelvin: %.2lf\n", celsius_temp, kelvin_temp);
    return 0;
}
